src/main/java/de/tum/in/www1/artemis/service/compass/CompassCalculationEngine.java

```diff
@@ -267,6 +269,12 @@ public void markModelAsUnassessed(long modelSubmissionId) {
                 continue;
             }
 
+            double elementConfidence = getConfidenceForElement(jsonElementID, modelId);
+            if (elementConfidence < ELEMENT_CONFIDENCE_THRESHOLD) {
+                log.error("Confidence " + elementConfidence + " of element " + jsonElementID + " is smaller than configured confidence threshold " + ELEMENT_CONFIDENCE_THRESHOLD);
```

Comment:
can we change this to log.debug? Or is it a serious problem? Otherwise this might spam the server logs

Reply:
Good catch, I changed it to `log.debug`.